src/doc/language.ts

```diff
@@ -678,26 +738,30 @@ export const doc_language:Documentation = {
 			<div class="example">
 				<h3>Example</h3>
 				<tscript>
-					class Circle
-					{
-						use namespace math;
+class Circle
+{
+    use namespace math;
 
-					private:
-						var m_radius = 0;
+	private:
+		var m_radius = 0;
 
-					public:
-						constructor(radius)
-						{ m_radius = radius; }
-						function radius()
-						{ return m_radius; }
-						function area()
-						{ return pi() * m_radius * m_radius; }
-					}
+    public:
+        # constructor() {                             # standard constructor, gets called by "var c = Circle()" and sets m_radius by default to 1
```

Comment:
Not sure if a commented constructor is helpful. Instead, use a default value for the radius?